### PR TITLE
feat: schema registry added to cloud services request [WIP]

### PR DESCRIPTION
**Pull request by secondsun**

### CODE REVIEW

File: olm/olm-template/manifests/rhoas-operator.cloudservicesrequests.crd.yaml [hunk 1]
```diff
@@ -84,5 +84,25 @@ spec:
                         type: string
                       status:
                         type: string
+                schemaRegistries:
```
Comment (wtrocki) [inline olm/olm-template/manifests/rhoas-operator.cloudservicesrequests.crd.yaml:87]: This will be breaking change so version should be changed

Comment (secondsun) [reply olm/olm-template/manifests/rhoas-operator.cloudservicesrequests.crd.yaml:87]: I think you can add without breaking things; I was going to confirm this in the k8s docs.

**Comment by wtrocki** [comment]
@secondsun note that there is already functional service registry example in guides

**Comment by secondsun** [comment]
@wtrocki  Ok, I can test rebase and merge this and then spinoff the SBO binding updates to quarkus into their own issue. Sound like a plan?

**Comment by wtrocki** [comment]
I think we have discussed that we going to keep development on the fork to be able to release operator without service registry support

**Comment by secondsun** [comment]
Closing in favor of feature branch.

**Pull request closed without merging**